easybuild/easyblocks/a/amber.py

```diff
@@ -213,10 +224,11 @@ def configuremake_install_step(self):
 
         # define environment variables for MPI, BLAS/LAPACK & dependencies
         mklroot = get_software_root('imkl')
-        openblasroot = get_software_root('OpenBLAS')
+        flexiblas_root = get_software_root('FlexiBLAS')
+        openblas_root = get_software_root('OpenBLAS')
         if mklroot:
             env.setvar('MKL_HOME', os.getenv('MKLROOT'))
-        elif openblasroot:
+        elif flexiblas_root or openblas_root:
             lapack = os.getenv('LIBLAPACK')
             if lapack is None:
                 raise EasyBuildError("LIBLAPACK (from OpenBLAS) not found in environment.")
```

Comment:
Should we change this error to mention the correct BLAS? i.e. FlexiBLAS or OpenBLAS?

Reply:
ping @boegel

Reply:
@akesandgren fixed in effecb7c1